gradle/libs.versions.toml

```diff
@@ -83,7 +83,7 @@ managed-micronaut-cache = "3.0.0"
 managed-micronaut-cassandra = "4.0.0"
 managed-micronaut-coherence = "3.0.1"
 managed-micronaut-data = "3.2.2"
-managed-micronaut-discovery = "3.0.0"
+managed-micronaut-discovery-client = "3.1.0"
```

Comment:
This is a potentially breaking change: if someone uses the `micronaut.discovery` property in their POM file, or `libs.micronaut.discovery` (in Gradle, unlikely since we didn't promote the feature). Not necessarily blocking, but wanted to mention this.

Reply:
@graemerocher Can we have both until 4.0?

Reply:
yeah probably makes sense